inst/startup/worker.R

```diff
@@ -71,7 +72,24 @@ setwd(batchTaskWorkingDirectory)
 
 azbatchenv <-
   readRDS(paste0(batchJobPreparationDirectory, "/", batchJobEnvironment))
-argsList <- azbatchenv$argsList
+
+tryCatch({
+  print("Data Set Boolean: ")
+
+  print(isDataSet)
+  print(typeof(isDataSet))
+  if (isDataSet) {
+    argsList <- readRDS(batchTaskEnvironment)
+  } else {
+    argsList <- azbatchenv$argsList
+    argsList <- argsList[startIndex:endIndex]
+  }
+
+  print(argsList)
```

Comment:
nitpick: Add something before printing off argsList so we know what this is.

Reply:
This was for debugging purposes. Removed